openadapt/events.py

```diff
@@ -103,31 +134,38 @@ def make_parent_event(child, extra=None):
     return models.ActionEvent(**event_dict)
 
 
-# Set by_diff_distance=True to compute distance from mouse to screenshot diff
-# (computationally expensive but keeps more useful events)
-def merge_consecutive_mouse_move_events(events, by_diff_distance=False):
-    """Merge consecutive mouse move events into a single move event"""
+def merge_consecutive_mouse_move_events(
+    events: list, by_diff_distance: bool = False
+) -> list:
```

Comment:
Should `list` be `list[ActionEvent]`?

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507